config/settings.yml

```diff
@@ -68,7 +68,7 @@ evss:
     timeout: 30
   mock_letters: false
   mock_gi_bill_status: false
-  mock_pciu: false
+  mock_pciu: true
```

Comment:
What about renaming to `mock_pciu_email`?  PCIU is a service made of of many endpoints, and email is just one of them.  It also falls in line with the `mock_pciu_address` precedent.

Reply:
@omgitsbillryan - Good question.  There was [a discussion around this](https://github.com/department-of-veterans-affairs/vets-api/pull/1718#discussion_r170157719) between @kreek & @jkassemi in the email PR, that resolved to that falling into a follow up refactoring story/PR, where `pciu_address` might be brought into and underneath the new `pciu` directory/service.

Will that still work?

Reply:
👍  with the goal of moving PCIU address into PCIU, since that's the logical boundary for the service that EVSS is presenting.